addons/settings/fnc_init.sqf

```diff
@@ -166,7 +166,7 @@ GVAR(default) setVariable [_setting, [_defaultValue, _setting, _settingType, _se
 private _settingInfo = GVAR(userconfig) getVariable _setting;
 
 if (isNil "_settingInfo") then {
-    private _settingsHash = profileNamespace getVariable [QGVAR(hash), HASH_NULL];
+    private _settingsHash = GET_SERVER_NAMESPACE getVariable [QGVAR(hash), HASH_NULL];
```

Comment:
clients run this as well, don't want them to use the GVAR(volatile)

Reply:
Fixed in [18f785d](https://github.com/CBATeam/CBA_A3/pull/1617/commits/18f785d1470f28e1013daa50c7e0035cc2b53002). Only dedicated will use `uiNamespace`